frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -179,12 +180,14 @@ const DeviceUserPage = ({
         org_contact_url,
         global_config,
         host: responseHost,
+        self_service,
       }) => {
         setShowRefetchSpinner(isRefetching(responseHost));
         setIsPremiumTier(license.tier === "premium");
         setOrgLogoURL(org_logo_url);
         setOrgContactURL(org_contact_url);
         setGlobalConfig(global_config);
+        setSelfService(self_service);
```

Comment:
Since we're currently setting states like this here this looks good for now. Ultimately all of these states are redundant since `useQuery` is holding this data in state internally already. I'll add this to frontend's code improvement to-dos.